### PR TITLE
Update the vnext branch

### PR DESCRIPTION
**Pull request by JoeRobich**

### CODE REVIEW

**Comment by filipw** [comment]
"Failure while loading the analyzer reference 'StyleCop.Analyzers': Assembly with same name is already loaded"

does SDK now ship stylecop by default?

**Comment by JoeRobich** [comment]
> does SDK now ship stylecop by default?
@filipw It does not. However, I just realized I'm targeting the wrong branch here.

**Comment by JoeRobich** [comment]
We needed as similar set of changes to analyzer assembly loading as Roslyn made in https://github.com/dotnet/roslyn/pull/55098

File: src/OmniSharp.Host/Services/DefaultAnalyzerAssemblyLoader.Desktop.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if !NETCOREAPP
```
Comment (filipw) [inline src/OmniSharp.Host/Services/DefaultAnalyzerAssemblyLoader.Desktop.cs:5]: I think maybe we will just drop desktop builds from the branch altogether?

**Review by filipw — APPROVED** [review]
thanks for keeping this dream alive

**Comment by filipw** [comment]
> We needed as similar set of changes to analyzer assembly loading as Roslyn made in [dotnet/roslyn#55098](https://github.com/dotnet/roslyn/pull/55098)

this is a very clever solution indeed, nice

**Pull request merged**